python/Ganga/Core/GangaRepository/VStreamer.py

```diff
@@ -192,8 +195,7 @@ def simpleAttribute(self, node, name, value, sequence):
                 self.level += 1
                 print(file=self.out)
                 print(self.indent(), '<sequence>', file=self.out)
-                for v in value:
-                    self.acceptOptional(v)
+                map(self.acceptOptional, value)
```

Comment:
As above, `map` is a functional programming construct for creating new sequences out of existing ones (like a list comprehension). It is not just a replacement for a `for` loop.